README.md

```diff
@@ -3,5 +3,9 @@
 # elastic-serverless-forwarder
 Elastic Serverless Forwarder
 
+## Important - v1.6.0
```

Comment:
just realised this is not the readme published on SAR: rather the one at https://github.com/elastic/elastic-serverless-forwarder/blob/main/docs/README-AWS.md